site/docs/components/accordion/usage.mdx

```diff
@@ -34,4 +34,18 @@ import {
 
 ### Props
 
+#### Accordion
+
 <PropsTable componentName="Accordion" />
+
+#### Accordion Group
+
+<PropsTable componentName="AccordionGroup" />
+
+#### Accordion Header
+
+<PropsTable componentName="AccordionHeader" />
+
+#### Accordion Panel
+
+<PropsTable componentName="AccordionPanel" />
```

Comment:
```suggestion

#### Accordion Group

<PropsTable componentName="AccordionGroup" />

#### Accordion Header

<PropsTable componentName="AccordionHeader" />

#### Accordion Panel

<PropsTable packageName="core" componentName="AccordionPanel" />
```

Reply:
Don't think this is required as core is default